contracts/guard/src/GuardV0.sol

```diff
@@ -17,6 +17,22 @@ import "./IGuard.sol";
  */
 contract GuardV0 is IGuard, Ownable {
     using Path for bytes;
+    using BytesLib for bytes;
+
+    /// @dev The length of the bytes encoded address
```

Comment:
Can you add a link to the source of these figures?

(Probably some 1delta source code?)